javaparser-core-testing/src/test/java/com/github/javaparser/JavaParserTest.java

```diff
@@ -308,4 +308,13 @@ void parseTypeDeclaration() {
     void xxx(){
         YamlPrinter.print(StaticJavaParser.parse("class X{}"));
     }
+
+    @Test
+    void issue2879() {
```

Comment:
Should this be in `JavaParserTest`? Perhaps we could move it to `MethodDeclarationTest` or similar?

Reply:
I didn't put it in `MethodDeclarationTest` because that file contains no calls to `StaticJavaParser.parse`, only calls to `StaticJavaParser.parseBodyDeclaration`.

I put it here becasue it is adjacent to a different call to `StaticJavaParser.parse`.

I didn't find a README, nor class documentation, explaining the purpose of each test directory and file.

Just let me know where you want it, and I will move it.

Reply:
> I didn't find a README, nor class documentation, explaining the purpose of each test directory and file.

Can't argue there -- very much a free-for-all! ;)